RELEASING.md

```diff
@@ -2,45 +2,18 @@
 
 This page describes how to cut releases of our plugin. It's strictly for the owners of the project.
 
-1. Create a new branch as if you were starting a normal bugfix PR
+1. Create a new branch named release_v{$version}.  It's critical that you create the branch according to this schema or the release will fail.
 
```

Comment:
Is there concern local branches wont be up to date?

how about something like?

```
git fetch
git checkout -b release_v1.0 origin/master
```